docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -0,0 +1,212 @@
+
+For the first 30 second of the Container's life, there is `/tmp/healthy` file.
+So during the first 30 seconds, the command `cat /tmp/healthy` returns a success
+code. After 30 seconds, `cat /tmp/healthy` returns a failure code.
+
+Create the Pod:
+
+    kubectl create -f http://k8s.io//docs/tasks/configure-pod-container/exec-liveness.yaml
+
+Within 30 seconds, view the Pod events:
+
+    kubectl describe pod liveness-exec
+
+The output indicates that no liveness probes have failed yet:
+
+    FirstSeen    LastSeen    Count   From            SubobjectPath           Type        Reason      Message
+    --------- --------    -----   ----            -------------           --------    ------      -------
+    24s       24s     1   {default-scheduler }                    Normal      Scheduled   Successfully assigned liveness-exec to worker0
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "gcr.io/google_containers/busybox"
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "gcr.io/google_containers/busybox"
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
+
+After 30 seconds, view the Pod events again:
+
+    kubectl describe pod liveness-exec
+
+At the bottom of the output, there are messages indicating that the liveness
+probes have failed, and the containers have been killed and recreated.
+
+    FirstSeen LastSeen    Count   From            SubobjectPath           Type        Reason      Message
+  --------- --------    -----   ----            -------------           --------    ------      -------
+    37s       37s     1   {default-scheduler }                    Normal      Scheduled   Successfully assigned liveness-exec to worker0
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
+    2s        2s      1   {kubelet worker0}   spec.containers{liveness}   Warning     Unhealthy   Liveness probe failed: cat: can't open '/tmp/healthy': No such file or directory
+
+Wait another minute, and verify that the Container has been restarted:
```

Comment:
Changed to 30 seconds. For me, it took about 30 seconds before **kubectl get pod** showed an incremented **RESTARTS**.